transaction-view/src/instructions_frame.rs

```diff
@@ -17,6 +17,15 @@ pub(crate) struct InstructionsFrame {
     pub(crate) num_instructions: u16,
     /// The offset to the first instruction in the transaction.
     pub(crate) offset: u16,
+    pub(crate) cached_offsets_and_lens: Vec<InstructionOffsetAndLens>,
+}
+
+#[derive(Debug)]
+pub(crate) struct InstructionOffsetAndLens {
+    num_accounts: u16,
+    accounts_offset: u16,
+    data_len: u16,
+    data_offset: u16,
```

Comment:
experimented with storing `u8`s for the lengths of the u16s, which saves 2 bytes per instruction.
benchmark results showed minor performance loss though - (103ktps -> 100ktps)